src/containers/MainPage/MainPage.js

```diff
@@ -2,7 +2,6 @@ import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { Container, Grid, Header } from 'semantic-ui-react';
```

Comment:
see if you can use separate import's for components like
```import Container from 'semantic-ui-react/Container';```

Reply:
For these import, I directly write the component in the code and then `vscode` imports for me.

Reply:
VsCode does that, but we need to import it separately so that the production build bundle size will be small and it wont contain code which we are not using like semantic-ui-react might provide more components if you import it like this webpack will pull all the code and bundle it.

Reply:
I searched about importing single components, but I don't think Semantic-UI-React provides that kind of importing of components.